From 2c300d471c0437c46f919f67e05628a1af37f36f Mon Sep 17 00:00:00 2001
From: Lin Huang <hl@rock-chips.com>
Date: Wed, 9 Jan 2019 17:29:22 +0800
Subject: [PATCH] arm64: rk1808_x4_linux_defconfig: enable
 CONFIG_REGULATOR_RK808

enable rk808 related config, so we can adjust voltage on rk1808x4
board.

Change-Id: I28ba4da20176a4e7f10afeb4e7c1389f4aa920ed
Signed-off-by: Lin Huang <hl@rock-chips.com>
---
 arch/arm64/configs/rk1808_x4_linux_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/configs/rk1808_x4_linux_defconfig b/arch/arm64/configs/rk1808_x4_linux_defconfig
index 1a2bf269163a..ae1b1d065d1c 100644
--- a/arch/arm64/configs/rk1808_x4_linux_defconfig
+++ b/arch/arm64/configs/rk1808_x4_linux_defconfig
@@ -112,10 +112,12 @@ CONFIG_DEVFREQ_THERMAL=y
 CONFIG_ROCKCHIP_THERMAL=y
 CONFIG_WATCHDOG=y
 CONFIG_DW_WATCHDOG=y
+CONFIG_MFD_RK808=y
 CONFIG_REGULATOR=y
 CONFIG_REGULATOR_FIXED_VOLTAGE=y
 CONFIG_REGULATOR_FAN53555=y
 CONFIG_REGULATOR_GPIO=y
+CONFIG_REGULATOR_RK808=y
 CONFIG_MEDIA_SUPPORT=y
 CONFIG_MEDIA_CAMERA_SUPPORT=y
 CONFIG_MEDIA_RC_SUPPORT=y
-- 
2.35.3

